### PR TITLE
Change: cv2 impath to abspath

### LINKED ISSUE #350: Change cv2 image path to absolute path

### PR DESCRIPTION
**Pull request by 12rics**
change cv2 impath to abspath and format document a little about import order, newline
Resolve #350

### CODE REVIEW

File: omnis/application/image_processing/object_detection/object_detection.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def data_generator(self, annotation_lines, batch_size,
         while True:
```
Comment (ik1ne) [inline omnis/application/image_processing/object_detection/object_detection.py]: Am I missing a part where the variable i is changed or is the loop works differently?
Also, even if I missed the variable modification it is better to use for loop if the iteration variable is apparent.

**Comment by ik1ne** [comment]
I added comment to one part where I wasn't sure about the logic.

Also, the change of import order that doesn't comply with alphabetical order. Is it intentional or by accident? Because I was ordering that according to [Google style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md), alphabetically.

File: omnis/application/image_processing/object_detection/object_detection.py (hunk 1 above)
Comment (12rics) [reply omnis/application/image_processing/object_detection/object_detection.py]: variable i is changed in line 238, and variable b is never used, and pylint warned about it.

**Comment by 12rics** [comment]
> I added comment to one part where I wasn't sure about the logic.
> 
> Also, the change of import order that doesn't comply with alphabetical order. Is it intentional or by accident? Because I was ordering that according to [Google style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md), alphabetically.

pylint warned about importing order, that cv2 has to be imported after numpy and keras.

**Comment by ik1ne** [comment]
```
(.venv) $ pylint --rcfile=./pylintrc omnis/application/image_processing/deblur/deblur.py                    
************* Module omnis.application.image_processing.deblur.deblur
omnis/application/image_processing/deblur/deblur.py:1:0: C0111: Missing module docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:22:0: C0103: Class name "Deblur_GAN" doesn't conform to '^_?[A-Z][a-zA-Z0-9]*$' pattern (invalid-name)
omnis/application/image_processing/deblur/deblur.py:22:0: C0111: Missing class docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:34:4: C0111: Missing method docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:56:4: C0111: Missing method docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:108:4: C0111: Missing method docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:159:4: C0111: Missing method docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:214:4: C0111: Missing method docstring (missing-docstring)
omnis/application/image_processing/deblur/deblur.py:231:4: C0111: Missing method docstring (missing-docstring)
```

Mine did not, can you check if you're using correct pylintrc file?

**Comment by 12rics** [comment]
> Mine did not, can you check if you're using correct pylintrc file?

I ran pylint again, and it doesn't show up warning like last time. Maybe pylint thought cv2 as a local application for some reason.

**Pull request closed without merging**